lib/flux/app-state.js

```diff
@@ -7,6 +7,25 @@ import analytics from '../analytics';
 
 const debug = Debug('appState');
 
+const toggleSystemTag = (note, systemTag, shouldHaveTag) => {
+  const {
+    data: { systemTags = [] },
+  } = note;
+  const hasTagAlready = systemTags.includes(systemTag);
+
+  return hasTagAlready !== shouldHaveTag
```

Comment:
This took me a while to parse.  No idea how it could be any simpler but it took me some time to figure out what was going on.

Reply:
yeah I don't know the best phrasing. This was the simplest I got.

```js
if ( noteIsAlreadyInAStateWhereThePresenceOrAbsenceOfTheTagMatchesTheExpectationOnceThisFunctionEnds )
```

```js
const tagIsAlreadyInNote = systemTags.includes(systemTag);
const shouldAddTagToNote = ?
```

In my previous work at least there was convention that `should_` indicates a desire to enact the thing and that it's boolean. Here in JS land we only have a single type `any` so the language doesn't help us out like it does in statically-typed languages.

Any recommendation or just noting that it's something that could be better some how?